AK/Variant.h

```diff
@@ -7,9 +7,9 @@
 #pragma once
 
 #include <AK/Array.h>
-#include <AK/BitCast.h>
 #include <AK/StdLibExtras.h>
 #include <AK/TypeList.h>
+#include <netinet/in.h>
```

Comment:
?

Reply:
how did that happen?
I blame clangd while I was not looking